vnext/Microsoft.ReactNative/ABIModule.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ABIModule.h"
+#include "NativeModulesProvider.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+auto ABIModule::getName() -> std::string {
+  auto name = m_module.Name();
+  return to_string(name);
```

Comment:
For my education, where is this 'to_string' method coming from?

Reply:
It's in the winrt namespace coming from the base.h that gets included.  I could have done [winrt::to_string](https://docs.microsoft.com/en-us/uwp/cpp-ref-for-winrt/to-string).